packages/ketcher-react/src/script/ui/views/components/MonomerCreationWizard/MonomerCreationWizard.tsx

```diff
@@ -329,6 +330,9 @@ const MonomerCreationWizard = () => {
   const { values, notifications, errors } = wizardState;
   const { type, symbol, name, naturalAnalogue } = values;
   const [modificationTypes, setModificationTypes] = useState<string[]>([]);
+  const [showLeavingGroupDialog, setShowLeavingGroupDialog] = useState(false);
+  const [leavingGroupDialogMessage, setLeavingGroupDialogMessage] =
```

Comment:
I suppose having two states here is unnecessary, perhaps you could just rely on non-empty message as a flag to display a dialog?